pypeit/scripts/tellfit.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+from pypeit import par, msgs
+from astropy.io import fits
+from pypeit.par import pypeitpar
+from pypeit.spectrographs.util import load_spectrograph
+import argparse
+from pypeit.core import telluric
+import os
+from pkg_resources import resource_filename
+
+
+# A trick from stackoverflow to allow multi-line output in the help:
+#https://stackoverflow.com/questions/3853722/python-argparse-how-to-insert-newline-in-the-help-text
+class SmartFormatter(argparse.HelpFormatter):
+
+    def _split_lines(self, text, width):
+        if text.startswith('R|'):
+            return text[2:].splitlines()
+        # this is the RawTextHelpFormatter._split_lines
+        return argparse.HelpFormatter._split_lines(self, text, width)
+
+def read_tellfile(ifile):
+    """
+    Read a PypeIt telluric file, akin to a standard PypeIt file
+
+    The top is a config block that sets ParSet parameters
+      The spectrograph is not required
+
+    Args:
```

Comment:
this is a mix of `numpy` and `Google` doc formatting

Pick one

Reply:
done